docs/messaging/messaging-spans.md

```diff
@@ -207,6 +207,7 @@ The following operations related to messages are defined for these semantic conv
 | `create`       | A message is created. "Create" spans always refer to a single message and are used to provide a unique creation context for messages in batch publishing scenarios. |
 | `receive`      | One or more messages are requested by a consumer. This operation refers to pull-based scenarios, where consumers explicitly call methods of messaging SDKs to receive messages. |
 | `deliver`      | One or more messages are passed to a consumer. This operation refers to push-based scenarios, where consumer register callbacks which get called by messaging SDKs. |
+| `process`      | One or more messages are processed by consumer. |
```

Comment:
should we also specify a kind (Server or Internal)?

Reply:
Added. After the operation is unified with "deliver", it needs to be `CONSUMER` for push-based scenarios.